src/components/reusable/overflowMenu/overflowMenu.scss

```diff
@@ -16,16 +16,16 @@ button {
   border-radius: 4px;
   cursor: pointer;
   color: inherit;
-  outline: 2px solid transparent;
+  outline: 1px solid transparent;
```

Comment:
I don't think we want to change the focus outline thickness.

Reply:
@brian-patrick-3 , changed to 2px